src/Avalonia.Base/Input/InputElement.cs

```diff
@@ -388,6 +414,25 @@ public bool IsEnabled
             get { return GetValue(CursorProperty); }
             set { SetValue(CursorProperty, value); }
         }
+
+        /// <summary>
+        /// Gets or sets a value that determines whether the Holding event can originate
+        /// from that element.
+        /// </summary>
+        public bool IsHoldingEnabled
+        {
+            get { return GetValue(IsHoldingEnabledProperty); }
+            set { SetValue(IsHoldingEnabledProperty, value); }
+        }
+
+        /// <summary>
+        /// Enables or disables support for the press and hold gesture through the left button on a mouse.
+        /// </summary>
+        public bool IsHoldWithMouseEnabled
```

Comment:
Not really important, but these two properties do the same, just for different input types. Feels like it can be combined with an enum, but I don't have good ideas for possible values.

Reply:
`IsHoldWithMouseEnabled` allows mouse pointers to trigger Hold. If `IsHoldingEnabled` is false, the hold gesture won't be detected at all, even if `IsHoldWithMouseEnabled` is true. This property is there to prevent context actions for being triggered from holding the mouse button, esp. in controls that allow repeated clicks from holding a button.